lang/cs/Org.Apache.REEF.Client/API/JobParameters.cs

```diff
@@ -68,5 +86,24 @@ public int DriverMemoryInMB
         {
             get { return _driverMemory; }
         }
+
+        /// <summary>
+        /// Gets the file path for stdout for the driver.
+        /// </summary>
```

Comment:
Whoops, sorry, I meant to remove that comment entirely, because it turned out not to be true -- I was just entering the paths incorrectly when I tried to test it out. I'll remove the message in the other places. Thanks!